pkg/plugins/runtime/k8s/webhooks/defaulter_test.go

```diff
@@ -155,7 +180,62 @@ var _ = Describe("Defaulter", func() {
               }
             }
 `,
-			expectedPatch: ``,
+			expected: `
+            {
```

Comment:
Is this right? `mesh: demo` but `kuma.io/mesh: default`?

Reply:
Yeah, it just ignores `mesh` field because new policies won't have `mesh` field. I deleted `mesh: demo` from here to avoid confusion in the future